riotfile.py

```diff
@@ -87,7 +87,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
     },
     venvs=[
         Venv(
-            pys=["3"],
+            pys=["3.9"],
```

Comment:
why this change?

Reply:
actually, are any of the changes to this file intended?

Reply:
Yes it is, because circleCI pipeline has configured py3.9 as default python version:
```
#.circleci/config.yml
  steps:
    - &pyenv-set-global
      run:
        name: Set global pyenv
        command: |
          pyenv global 3.9.4
```
You could have different results with pre-commit and hooks if you run with different python version (I checked it with py3.8 and py3.10)

Reply:
can you open these changes in a different PR since they seem unrelated to the current PR?

I'd like to learn more about the specific issue since I don't think any of us have run into it yet?